CMS.drawio

```diff
@@ -0,0 +1 @@
 style="text;strokeColor=none;fillColor=none;align=left;verticalAlign=top;spacingLeft=4;spacingRight=4;overflow=hidden;rotatable=0;points=[[0,0.5],[1,0.5]];portConstraint=eastwest;" parent="44" vertex="1"><mxGeometry y="26" width="140" height="26" as="geometry"/></mxCell><mxCell id="47" value="https://cms.reearth.io/api/v1/projects/{project.alias}/models/{model.key}/items" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" parent="1" vertex="1"><mxGeometry x="10" y="66" width="420" height="20" as="geometry"/></mxCell><mxCell id="50" value="need to check again! If we allow to specify user or role?" style="text;strokeColor=none;fillColor=none;align=left;verticalAlign=top;spacingLeft=4;spacingRight=4;overflow=hidden;rotatable=0;points=[[0,0.5],[1,0.5]];portConstraint=eastwest;" parent="1" vertex="1"><mxGeometry x="890" y="40" width="320" height="26" as="geometry"/></mxCell><mxCell id="51" value="Comment" style="swimlane;fontStyle=0;childLayout=stackLayout;horizontal=1;startSize=26;fillColor=none;horizontalStack=0;resizeParent=1;resizeParentMax=0;resizeLast=0;collapsible=1;marginBottom=0;" parent="1" vertex="1"><mxGeometry x="960" y="-160" width="140" height="130" as="geometry"/></mxCell><mxCell id="52" value="requestID" style="text;strokeColor=none;fillColor=none;align=left;verticalAlign=top;spacingLeft=4;spacingRight=4;overflow=hidden;rotatable=0;points=[[0,0.5],[1,0.5]];portConstraint=eastwest;" parent="51" vertex="1"><mxGeometry y="26" width="140" height="26" as="geometry"/></mxCell><mxCell id="53" value="commenterID" style="text;strokeColor=none;fillColor=none;align=left;verticalAlign=top;spacingLeft=4;spacingRight=4;overflow=hidden;rotatable=0;points=[[0,0.5],[1,0.5]];portConstraint=eastwest;" parent="51" vertex="1"><mxGeometry y="52" width="140" height="26" as="geometry"/></mxCell><mxCell id="54" value="message" style="text;strokeColor=none;fillColor=none;align=left;verticalAlign=top;spacingLeft=4;spacingRight=4;overflow=hidden;rotatable=0;points=[[0,0.5],[1,0.5]];portConstraint=eastwest;" parent="51" vertex="1"><mxGeometry y="78" width="140" height="26" as="geometry"/></mxCell><mxCell id="55" value="createdAt" style="text;strokeColor=none;fillColor=none;align=left;verticalAlign=top;spacingLeft=4;spacingRight=4;overflow=hidden;rotatable=0;points=[[0,0.5],[1,0.5]];portConstraint=eastwest;" parent="51" vertex="1"><mxGeometry y="104" width="140" height="26" as="geometry"/></mxCell><mxCell id="58" style="edgeStyle=none;html=1;endArrow=ERmany;endFill=0;" parent="1" source="49" target="52" edge="1"><mxGeometry relative="1" as="geometry"/></mxCell><mxCell id="61" value="https://cms.reearth.io/api/v1/{project.alias}/{model.key}/items" style="text;html=1;align=left;verticalAlign=middle;resizable=0;points=[];autosize=1;strokeColor=none;fillColor=none;" parent="1" vertex="1"><mxGeometry x="10" y="90" width="340" height="20" as="geometry"/></mxCell></root></mxGraphModel></diagram></mxfile>
```

Comment:
These draw.io files tend to be out of date and are not recommended for inclusion in the repository. As a general rule, the source code and comments should explain this.